dsl/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
You can ignore all `build` directories everywhere by having `build/` (`/` is just for being explicit) in root `.gitignore` :)